Gifski/MainWindowController.swift

```diff
@@ -320,8 +337,38 @@ extension MainWindowController: NSMenuItemValidation {
 		switch menuItem.action {
 		case #selector(open)?:
 			return !isRunning
+		case #selector(quickLook(_:))?:
+			return conversionCompleted
 		default:
-			return validateMenuItem(menuItem)
+			return true
+		}
+	}
+}
+
+extension MainWindowController: QLPreviewPanelDataSource {
```

Comment:
Wouldn't this logic fit better in ConversionCompletedView.swift?

Reply:
Yeah, I had a problem with that and decided it was fine to leave it in the window controller as it was a general advise I've read. But, I've did some research on why it wasn't working and it turns out that the responder chain got confused and didn't have the conversion view in it. I've managed to hack my way around it (I'm gonna make a comment on the line), but I'm not sure this is the best solution. Was researching other people's implementation on GH and SO but didn't see this type of use-case with main menu + QL preview, which is odd.